### ISSUE TITLE
Progress bar to reflect annotated images in a task

### ISSUE THREAD

**Issue by MahdiEsf** [opencv/cvat#1369]
Would be great to track the progress of annotators working on their assigned tasks via the **progress bar** on the main page where tasks are listed. As of now, the progress bar doesn't move as images are annotated.

My idea is to measure the progress by images with at least one label created for. In case of multiple labels and depending on applications, not all labels could be present in all images but should have at least one to count toward the progress metric. @nmanovic

**Comment by nmanovic** [comment]
@MahdiEsf , internally for tracking our progress we try to estimate number of objects in a data annotation task. The estimation isn't precise but usually give a good sense of the current status. 

[xxxxxxxxxx____________________]
The progress should show a precise number (it is impossible to calculate) but can have status: "less than 25%, less than 50%, less than 75%, less than 100%". Also it is possible to calculate the approximate amount of time in working days. 

To make the calculation the user should provide the estimated number of objects.

**Comment by thomasstats** [comment]
Yeah, before switching to CVAT our internal marking system we used simply checked to see if there was at least 1 annotation on an image to add it to the progress bar.  Since switching to CVAT the only complaint I've really heard is the annotators saying they aren't as motivated because they have no progress bar.

**Comment by nmanovic** [comment]
OK. Probably, it will be great to make a feature to set the progress manually, and after that, use different algorithms to automate it. I agree that a common understanding of the progress is very important. But counting it in the suggested way will not work for many projects.
